app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -25,6 +27,16 @@ def index
         end
       end
 
+      # Returns all pages as a tree from the root given by the id parameter
+      #
+      def tree
+        authorize! :tree, :alchemy_admin_pages
+
+        render json: PageTreeSerializer.new(@page, ability: current_ability,
+                                            user: current_alchemy_user,
+                                            full: params[:full] == 'true')
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.